merlin/systems/triton/models/executor_model.py

```diff
@@ -119,7 +119,8 @@ def execute(self, requests):
                 tb_string = repr(traceback.extract_tb(exc_traceback))
                 responses.append(
                     pb_utils.InferenceResponse(
-                        tensors=[], error=f"{exc_type}, {exc_value}, {tb_string}"
+                        output_tensors=[],
```

Comment:
This change corrects the arguments passed to `InferenceResponse` which makes the error messages a bit clearer when something goes wrong. Previously errors were all wrapped in a message about arguments to the `InferenceResponse` which made things more difficult to quickly scan and see what was going wrong:

```
Failed to process the request(s) for model instance '1_predictforest', message: TypeError: __init__(): incompatible constructor arguments. The following argument types are supported:
E           1. c_python_backend_utils.InferenceResponse(output_tensors: List[c_python_backend_utils.Tensor], error: c_python_backend_utils.TritonError = None)
E
E       Invoked with: kwargs: tensors=[], error="'...
```

### Example

**Before**

![image](https://user-images.githubusercontent.com/1216955/198388322-905c42bc-3d5b-44e1-af86-49212fff353f.png)


**After**

![image](https://user-images.githubusercontent.com/1216955/198388640-a7b91746-c1aa-4749-bede-fa32fb5076f5.png)